apps/gdevelop/gdevelop.yml

```diff
@@ -0,0 +1,25 @@
+name: 'GDevelop'
+description: 'Create games without programming.'
+website: 'https://gdevelop-app.com/'
+repository: 'https://github.com/4ian/GDevelop'
+keywords:
+  - 'Game Editor'
+  - 'Open Source'
+  - 'Cross-Platform '
+  - 'Visual Programming'
+  - 'Online Editor'
+  - 'Piskel Sprite Editor'
+  - 'Google AdMob'
+license: MIT
+category: 'Graphics & Design'
+npmPackageName: 'gdmod'
+youtube_video_url: 'https://www.youtube.com/watch?v=5jdOR-NAiSA'
+screenshots:
+  - imageUrl: 'https://github.com/4ian/GDevelop/blob/master/newIDE/gd-ide-screenshot.png'
+    caption: 'GDevelop Editor Dark Mode'
+  - imageUrl: 'https://github.com/4ian/GDevelop/blob/master/newIDE/integrated-ide-screenshot.png'
```

Comment:
This is an outdated screenshot, can you remove it?

Reply:
Yea i thought so too, I ll remove it right away!

Reply:
Should I remove both the dark version & the light version screenshot or just the light Version?

Reply:
`https://github.com/4ian/GDevelop/blob/master/newIDE/gd-ide-screenshot.png` is not outdated, you can keep it. You can use `https://img.itch.zone/aW1nLzE1ODkwOTIucG5n/original/nP4Tp%2F.png` to showcase "Piskel sprite editor".

Reply:
(See https://4ian.itch.io/gdevelop for other nice images)

Reply:
Alright, Though I think 3 are enough! If you have any suggestions then I'll add them too.

Reply:
May I add them?

Reply:
https://img.itch.zone/aW1nLzE2MDEyNDcucG5n/original/kS6eis.png is more adapted to a website while there it's supposed to be screenshot of the software.
https://img.itch.zone/aW1nLzE1OTMzNzAucG5n/original/g7gty6.png might be good to give an example of a game.

Reply:
I 'll add the latter then.

Reply:
What should I keep the caption for https://img.itch.zone/aW1nLzE1OTMzNzAucG5n/original/g7gty6.png, should it be "Example Of a Game developed in the Editor"